public/javascripts/SVLabel/src/SVLabel/navigation/MapService.js

```diff
@@ -587,6 +586,19 @@ function MapService (canvas, neighborhoodModel, uiMap, params) {
         return uiMap.pano.find('svg').parent();
     }
 
+    /**
+     * 
```

Comment:
It looks like there's a lot of extra whitespace here that you don't need. I'd remove lines 599, 594, 596, and 598.